docs/angular/api-angular/generators/scam.md

```diff
@@ -1,6 +1,6 @@
 # @nrwl/angular:scam
 
-Generate a component with an accompanying Single Angular Module Component (SCAM).
+Generate a component with an accompanying Single Component Angular Module (SCAM).
```

Comment:
How did we miss this? 🤦🏻

Reply:
I have no idea lol noticed it yesterday when I read the deployed docs page